src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -1,20 +1,11 @@
 echo off
 setlocal
 
-echo BridgeKeepRunning=%BridgeKeepRunning%
-if '%BridgeKeepRunning%' neq 'true' (
-    echo Releasing Bridge resources and stopping it if running locally.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
-    call Bridge.exe -stopIfLocal -reset %*
-    popd
-) else (
-    echo Releasing Bridge resources but leaving it running.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -reset %* ...
-    call Bridge.exe -reset %*
-    popd
-    echo The Bridge was left running because BridgeKeepRunning is true
-)
+echo Releasing Bridge resources and stopping it if running locally.
+pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+echo Invoking Bridge.exe -stopIfAutoStart -reset %* ...
+call Bridge.exe -stopIfAutoStart -reset %*
```

Comment:
Regarding remembering "StartBridge" means "manually started and won't be stopped by tests" and "EnsureBridge" means "automatically started and can be stopped by tests", that sounds like a matter of choosing the right name.  Suggestions to improve? Just "StartBridge" and "AutoStartBridge?"  "StartLongRunningBridge?"  Note that these CMD's are only wrappers around how Bridge.exe gets invoked.  The ToF test scripts will not use them because they directly invoke Bridge.exe (the ToF tests run in an environment where there is no source or script code, so the Bridge already needs to have been built).